app/components/utils/th-sortable.hbs

```diff
@@ -0,0 +1,14 @@
+<th class="auk-u-text-nowrap" role="columnheader" aria-sort="none" ...attributes >
+  <button class="auk-table-sorter " type="button" onclick={{@onChange}}>
+    <div class="auk-table-sorter__label {{if this.isSorted "auk-u-text-bold"}}">{{@label}}</div>
+    <div class="auk-icon auk-icon--default auk-icon--muted">
+      {{#if  (and this.isSorted (eq this.order 'desc'))}}
+        <i class="ki ki-caret-down"></i>
+      {{else if (and this.isSorted (eq this.order 'asc'))}}
+        <i class="ki ki-caret-up"></i>
```

Comment:
Voor deze logica zou ik een `get isSortedAscending()` en `get isSortedDescending()` definieren in de component om de template leesbaarder te maken.